app/code/core/Mage/Eav/etc/config.xml

```diff
@@ -132,6 +132,18 @@
             </modules>
         </translate>
     </frontend>
+    <crontab>
+        <jobs>
+            <clean_orphaned_records>
```

Comment:
```suggestion
            <eav_clean_orphaned_records>
```

All other module's cron events seem to prefix with module name, i.e. `core_clean_cache`, `customer_flowpassword`, `newsletter_send_all`, `sales_clean_quotes`.